website/docs/cdktf/create-and-deploy/configuration-file.mdx

```diff
@@ -101,6 +101,24 @@ When you declare providers in JSON, add the constraint in the `version` property
 }
 ```
 
+## Enable Features
+
+CDKTF uses feature flags to enable potentially breaking behaviors in a release. Flags are stored as Runtime context values in `cdktf.json`. Feature flags are disabled by default, so existing projects that do not specify the flag will continue to work as expected with later CDKTF releases. New projects created using `cdktf init` include flags enabling all features available in the release that created the project.
```

Comment:
```suggestion
CDKTF uses feature flags to enable potentially breaking behaviors in a release. Flags are stored as runtime context values in `cdktf.json`. Feature flags are disabled by default, so existing projects that do not specify the flag will continue to work as expected with later CDKTF releases. New projects created using `cdktf init` include flags enabling all features available in the release that created the project.
```
It was like this before, but I think there's no reason to capitalize it, is there?

Reply:
no! there is not :-) good catch - thank you!